### ISSUE TITLE
Investigate update to Tails 6

### ISSUE THREAD

**Issue by zenmonkeykstop** [freedomofpress/securedrop#7079]
A bookworm-based version of Tails may drop soon (bullseye EOL approacheth), so we need to:
- [x] determine (tentative) schedule for it
- [x] get access to beta/testing channel if it is available
- [x] scope changes to admin scripts (probably mostly relating to Python version bump)
- [x] #7092

**Comment by nathandyer** [comment]
I reached out to the Tails Foundation team via email last week, and received a very kind and generous reply earlier this week.

They shared a tentative release date for Tails 6 with us (in confidence, so I won't share it here), and welcomed our feedback as we test the early builds for the new release. They shared a link with us to a nightly builds directory, which _is_ public, so I will provide it here: https://nightly.tails.boum.org/build_Tails_ISO_feature-bookworm/lastSuccessful/archive/build-artifacts/

They also flagged that they are considering having external drives mount automatically, which _may_ have an impact on our workflow: https://gitlab.tails.boum.org/tails/tails/-/issues/15900

I did some testing earlier this week with the nightly build, but ran into issues in both installing the necessary debian packages (it kept spawning multiple apt processes that would hang, and keep the dpkg lock). Even after manually intervening and installing those packages, I was unable to get pip to download the dependencies, as it seemed to refuse the network connection. I plan to try again with an updated nightly build in the coming days.
